cmd/up.go

```diff
@@ -228,6 +240,15 @@ func (cmd *UpCmd) Run(
 		}
 	}
 
+	// if GpgAgentForwarding we need to keep running in order to keep the reverse tunnel
+	// functioning, or gpg-agent will drop
+	if cmd.GpgAgentForwarding {
+		log.Infof(
+			"GPG Agent forwarding specified, keep this process running to have working gpg-agent forwarding",
+		)
+		select {}
```

Comment:
Why not call this at the end of up and then block?

Reply:
Would also be good to have a context variable for this so we don't need to use the flag everytime